scripts/src/java/org/oppia/android/scripts/todo/IssueTodosCommentCheck.kt

```diff
@@ -29,6 +29,11 @@ fun main(vararg args: String) {
 
   val failureCommentFilePath = args[2]
 
+  val permaLinkPrefix = "https://github.com/oppia/oppia-android/blob/"
+
+  // Here, we are adding 40 to account for the commit Sha size
```

Comment:
```suggestion
  // Here, we are adding 40 to account for the commit SHA-1 hash length (for context: https://en.wikipedia.org/wiki/SHA-1).
```

This helps clarify the correctness behind the magic number.

Reply:
Done.